lifecycle-management/Ahk.Lifecycle.Management.DAL/Helper/NewtonsoftJsonCosmosSerializer.cs

```diff
@@ -0,0 +1,43 @@
+using System.Text;
+using Microsoft.Azure.Cosmos;
+using Newtonsoft.Json;
+
+namespace Ahk.Lifecycle.Management.DAL.Helper
+{
+    public class NewtonsoftJsonCosmosSerializer : CosmosSerializer
```

Comment:
Nem értem, erre mi szükség van, nincs benne semmi saját logika.

Reply:
A `CosmosSerializer` egy abstract osztály, minden beépített implementációja [sealed és internal](https://github.com/Azure/azure-cosmos-dotnet-v3/issues/1813), így nem tudom a beépítettnek átadni a convertereket.